client/vfm-react-app/src/App.js

```diff
@@ -10,16 +10,22 @@ import CustomerSearch from './pages/CustomerSearch';
 import FarmerSearch from './pages/FarmerSearch';
 import Register from './pages/Register';
 import Login from './pages/Login';
-
+import axios from "axios";
 
 function App() {
-  // We may want to move this out or make it static
-  let [user, setUser] = React.useState(null) // once logged in this should be set to current user
+  //Access current user id using following statements: localStorage.getItem('curr_user_id'), localStorage.setItem('curr_user_id', JSON.stringify(curr_user_id))
+
+  const [curr_user_id, setCurr_User_Id] = React.useState(
+    //temporarily commenting out below line
+    JSON.parse(localStorage.getItem('curr_user_id')) || 0
+  );
+
+  localStorage.setItem('curr_user_id', JSON.stringify(curr_user_id));
+
   return (
     <BrowserRouter>
       <Routes>
         <Route path="/" element={<LandingPage />} />
-        
         <Route path="/farmer" element={<FarmerLandingPage />} />
         <Route path="/customer" element={<CustomerLandingPage />} />
```

Comment:
For 1.0 we may want to consider changing the routes to be something like `/farmer/:userid`. And just pass the user id to the farmer landing page and customer landing page. (that way you wouldn't have to pull from local storage every time